Example/Stripe iOS Example (Simple)/CheckoutViewController.swift

```diff
@@ -13,13 +13,13 @@ class CheckoutViewController: UIViewController, STPPaymentContextDelegate {
 
     // 1) To get started with this demo, first head to https://dashboard.stripe.com/account/apikeys
     // and copy your "Test Publishable Key" (it looks like pk_test_abcdef) into the line below.
-    let stripePublishableKey = ""
+    let stripePublishableKey = "pk_test_4TDXAGLdZFGNbXYGajQlcstU"
```

Comment:
ah yeah, just left these here while developing – i'll remove them before the demo